eventuate-local-java-cdc-connector-unified/src/main/java/io/eventuate/local/unified/cdc/pipeline/common/configuration/CdcDefaultPipelinePropertiesConfiguration.java

```diff
@@ -29,6 +29,11 @@ public CdcPipelineProperties cdcPipelineProperties(DefaultSourceTableNameResolve
             .ofNullable(eventuateConfigurationProperties.getSourceTableName())
             .orElse(sourceTableNameResolver.resolve("default")));
 
+    cdcPipelineProperties.setPurgeMessagesEnabled(eventuateConfigurationProperties.getPurgeMessagesEnabled());
```

Comment:
Why?
A cleaner should know about a pipeline. But a pipeline should not know about a cleaner.

Reply:
ok, I will fix